tests/testthat/test-utils.R

```diff
@@ -557,4 +566,27 @@ test_that("capVals works as expected", {
   expect_equal(dt1c, dt2)
   
   expect_error(capVals(as.list(dt1)), "Must be a data.table")
-})
\ No newline at end of file
+})
+
+test_that("calc_sd returns correct standard deviation for vectors with length > 1", {
```

Comment:
Or maybe that coudl be one section - as all others function above?
```
test_that("cap calc_sd Vals works as expected", {
# calc_sd returns correct standard deviation for vectors with length > 1
  epect_equal(calc_sd(c(1, 2, 3, 4, 5)), sd(c(1, 2, 3, 4, 5), na.rm = TRUE))
  expect_equal(calc_sd(c(10, 20, 30)), sd(c(10, 20, 30), na.rm = TRUE))

# calc_sd returns 0 for vectors with length == 1 and numeric.
expect_equal(calc_sd(c(1)), 0)
...

})
```

Reply:
done